solidity/test/geyser/TestKeepTokenGeyser.js

```diff
@@ -371,24 +369,29 @@ describe("KeepTokenGeyser", async () => {
       })
 
       // Validate stakers' stake token balances.
-      expect(
+      expectCloseTo(
```

Comment:
Would it be possible to use `expect(...).to.be.closeTo(...)`, [here](https://github.com/keep-network/coverage-pools/blob/140403c6cc9bf3f93e39f4894075f09f28536cae/test/AssetPoolTest.js#L150-L155) is an example.

https://www.chaijs.com/api/bdd/#method_closeto

Reply:
`expect(...).to.be.closeTo(...)` expects params of type `Number`, here we've got `BN`.

When I pass the values as `BN` I get: `AssertionError: expected <BN: 1fc3842bd1f071c00000> to be a number`.
When I pass the values as `number` I get: `Error: Number can only safely store up to 53 bits`.

I'm not sure it's possible with truffle.